entrypoint.sh

```diff
@@ -10,4 +10,5 @@ gunicorn \
     --workers $GUNICORN_WORKERS \
     --threads $GUNICORN_THREADS \
     --timeout $GUNICORN_TIMEOUT \
+    --log-level debug \
```

Comment:
```suggestion
    --log-level $GUNICORN_LOG_LEVEL \
```
En ajoutant la variable là où il faut :)